### PR TITLE
[FEATURE] Add readonly to handlebars autocompletion

### LINKED ISSUE #80: Add "readonly" helper to auto completion
Just like the [`mut`](http://emberjs.com/api/classes/Ember.Templates.helpers.html#method_mut) helper, the (currently undocumented, but existing) `readonly` helper should be part of the [auto completion code](https://github.com/Turbo87/intellij-emberjs/blob/master/src/main/kotlin/com/emberjs/hbs/HbsCompletionContributor.kt).

### PR DESCRIPTION
**Pull request by Serabe**
Add `readonly` helper to autocompletion.

Fixes #80

### CODE REVIEW

File: src/main/kotlin/com/emberjs/hbs/HbsCompletionContributor.kt [hunk 1]
```diff
@@ -15,13 +15,13 @@ class HbsCompletionContributor : CompletionContributor() {
     init {
         extend(CompletionType.BASIC, SIMPLE_MUSTACHE_NAME_ID, HbsBuiltinHelperCompletionProvider(
                 "action", "component", "debugger", "get", "if", "input", "link-to", "loc", "log",
-                "outlet", "partial", "render", "textarea", "unbound"))
+                "outlet", "partial", "readonly", "render", "textarea", "unbound"))
```
Comment (Turbo87) [inline src/main/kotlin/com/emberjs/hbs/HbsCompletionContributor.kt]: is `{{readonly something}}` a valid use case? I thought it was similar to `mut` and you only use it inside of a subexpression like `{{some-component (readonly something)}}`

Comment (Serabe) [inline src/main/kotlin/com/emberjs/hbs/HbsCompletionContributor.kt]: Sorry, I totally misunderstood what those were for.

Comment (Turbo87) [inline src/main/kotlin/com/emberjs/hbs/HbsCompletionContributor.kt]: hehe, no problem. first is for `{{FOO}}`, second for `{{#FOO}}` and third for `{{foo (FOO)}}`.

**Comment by Turbo87** [comment]
thanks! 👍

**Pull request merged**